index.html

```diff
@@ -0,0 +1,58 @@
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css" integrity="sha512-u2Qo5c5wL2fZ5m0z/rqZjLmZ9+vHjE/AqY0rU/QdugnfeE0k8zO6rU2qzUITjtWc6i1Kc2nh/gPmKv5fK/xJwg==" crossorigin="anonymous" >
+
+  </head>
+  <body>
+    <div class="portfolio">
+        <header class="details">
+            <div class="name">
+                <span class="inside-name">Kelvin</span>
+            </div>
+            <nav>
+            <ul class="nav-list">
+                   <li><a href="#" class="item-home">Portfolio</a></li>
+                    <li><a href="#" class="item-about">About</a></li>
+                    <li><a href ="#"class="item-contact">Contact</a></li>
+                    <li><a href="#"class="item-email">
+                        <img src="images/Disabled.png"alt="Disabled">
+                    </a></li>
+                   
+                </ul>
+                </nav>
+               
+        </header>
+        <section class="section-container">
```

Comment:
- [ ] Please follow the style guidelines for this section.

## Your Design:
![image](https://user-images.githubusercontent.com/92215878/213515538-c6e76f50-4d66-440a-b40c-047be5d387c7.png)

## Figma Design:
![image](https://user-images.githubusercontent.com/92215878/213515653-b47d2bb4-7aaa-4ed8-8f00-2fc70ab05738.png)